lib/dmv.rb

```diff
@@ -1,15 +1,20 @@
 class Dmv
-
+attr_reader :facility, :service
```

Comment:
I'm curious about both of these attr_readers. I don't see any attributes named `@facility` or `@service`, so there doesn't seem to be a reson to have them.